curator-framework/src/main/java/org/apache/curator/framework/imps/CuratorFrameworkImpl.java

```diff
@@ -240,11 +238,7 @@ public void createContainers(String path) throws Exception
     @Override
     public void clearWatcherReferences(Watcher watcher)
     {
-        NamespaceWatcher namespaceWatcher = namespaceWatcherMap.remove(watcher);
-        if ( namespaceWatcher != null )
-        {
-            namespaceWatcher.close();
-        }
```

Comment:
Just a question here.  The super class doc reads `@deprecated As of ZooKeeper 3.5 Curators recipes will handle removing watcher references when they are no longer used.`

Is this still the most accurate comment?  My (loose) understanding is that even the recipes themselves don't need to worry about this as the framework will handle it for them.